x/earn/keeper/grpc_query.go

```diff
@@ -397,6 +397,28 @@ func (s queryServer) getOneAccountAllDeposits(
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
+	if req.ValueInStakedTokens {
```

Comment:
We *might* want to consider reducing this to a single depsoit address address or adding a hard pagination limit.  A N+(1 to many) query type situation with many a thousand or more deposits would get pretty slow and likely stall block processing

Reply:
This should be limited to only 1 account, the base query handler for Deposits requires a Depositor address

https://github.com/Kava-Labs/kava/blob/b40d923c13c06e3161a5f96c48f1cb218142956c/x/earn/keeper/grpc_query.go#L247-L249